lib/main.dart

```diff
@@ -23,6 +23,7 @@ class _PerkPageState extends State<PerkPage> {
   String selectedType = 'survivor/';
   bool isSwitched = false;
   bool perkArraySelector = true;
+  bool lightModeDisabled = false;
```

Comment:
I'm kinda thinking that dark mode should be the default (it looks better IMO).
I'm also thinking that we should be consistent with variable naming:
    We already have a variable called "enabled" for the perks - so we should stick to using "enabled" instead of "disabled", i.e "lightModeEnabled = true;"